src/urllib3/util/proxy.py

```diff
@@ -35,8 +40,12 @@ def connection_requires_http_tunnel(
 
 
 def create_proxy_ssl_context(
-    ssl_version, cert_reqs, ca_certs=None, ca_cert_dir=None, ca_cert_data=None
-):
+    ssl_version: Optional[int] = None,
+    cert_reqs: Optional[int] = None,
+    ca_certs: Optional[str] = None,
+    ca_cert_dir: Optional[str] = None,
+    ca_cert_data: Optional[str] = None,
```

Comment:
Can `ca_cert_data` be of type `bytes`? I'm not sure

Reply:
Right, typeshed uses `cadata: Union[Text, bytes, None] = ...`